mimic/model/clb_objects.py

```diff
@@ -183,20 +225,15 @@ def add_load_balancer(self, tenant_id, lb_info, lb_id, current_timestamp):
         if "lb_building" in self.meta[lb_id]:
             status = "BUILD"
 
-        # Add tenant_id and nodeCount to self.lbs
+        # Add tenant_id to self.lbs
         current_timestring = seconds_to_timestamp(current_timestamp)
-        self.lbs[lb_id] = load_balancer_example(lb_info, lb_id, status,
-                                                current_timestring)
+        self.lbs[lb_id] = CLB(load_balancer_example(lb_info, lb_id, status,
```

Comment:
Clean-up idea, possibly for the future: this method (as several in this module) does `self.lbs[lb_id]` a whole bunch of times; it might make sense to extract that into a local variable.